okapi/services/caches/search/all.xml

```diff
@@ -169,6 +169,14 @@
         will set this parameter to <b>true</b>, then you will <b>always</b>
         receive an empty result.</p>
     </opt>
+    <opt name='powertrail_id'>
+        <p>BETA. Pipe-separated list of ints. If set, only caches which belong to 
+        <b>active</b> Power Trail which id is given will be included.</p>
```

Comment:
BTW, dlaczego tylko active? Skoro umożliwiamy wyselekcjonowanie po konkretnych identyfikatorach, to osoba wołająca metodę prawdopodobnie wie co robi - tzn. być może celowo chce wybrać kesze z nieaktywnego powertraila? Chyba, że nieaktywne powertraile są "tajne"?